functions/dialogflow/firebase.json

```diff
@@ -0,0 +1,7 @@
+{
+  "functions": {
+    "predeploy": [
```

Comment:
Is this step necessary for this sample? Seems like a lot of overhead.

Reply:
I will remove these lines.